### ISSUE TITLE
Errors in documented return types

### ISSUE THREAD

**Issue by danielspk** [vlucas/phpdotenv#391]
Hello, I detect that there are errors in the documentation of the return of types in multiple classes. For example:

```php
// class:  Dotenv\Repository\RepositoryBuilder

/**
  * Create a new repository builder instance.
  *
  * @return void
  */
public static function create()
{
    return new self();
}
```

The return type must be self. Example:

```php
// class:  Dotenv\Repository\RepositoryBuilder

/**
  * Create a new repository builder instance.
  *
  * @return self
  */
public static function create()
{
    return new self();
}
```

This prevents analyzing my project with the latest version of PHPStan v0.12.

Is it something I would be willing to modify?

Regards!

**Comment by GrahamCampbell** [comment]
This has already been fixed in master I think. :)

**Comment by GrahamCampbell** [comment]
It will be included in v4.0.1 or v4.1.0, whichever comes next.
